src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -1261,9 +1261,9 @@ function Base.show(io::IO, H::Hyperplane)
     n = length(normal_vector(H))
     b = negbias(H)
     if iszero(b) && iszero(normal_vector(H))
-        print(io, "The trivial Hyperplane, R^$n")
+        print(io, "Trivial Hyperplane, R^$n")
```

Comment:
Please change "Hyperplane" to "hyperplane", and "Halfspace" to "half-space"

Reply:
Done.